pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.ecocode</groupId>
     <artifactId>ecocode-parent</artifactId>
-    <version>1.4.8-SNAPSHOT</version>
+    <version>1.4.9-SNAPSHOT</version>
```

Comment:
same review feddback as above, no need to change this version in a starndard use case
but that you can done is update to 1.5.0-SNAPHOST because of deletion of deprecated rules

Reply:
I chaged to 1.5.0